src/game.rs

```diff
@@ -1,22 +1,22 @@
-use macroquad::{
-    color::{BLACK, WHITE},
-    shapes::draw_rectangle,
-    window::{clear_background, next_frame},
-};
+use macroquad::prelude::*;
+
+const DEFAULT_SPEED: f32 = 10.0;
 
 /// Represents the state of the Game of Life.
 pub struct Game {
+    ncols: usize,
+    nrows: usize,
     cell_size: usize,
     cells: Vec<Vec<bool>>,
 }
 
 impl Game {
-    /// Creates a new Game of Life with a random initial state.
+    /// Creates a new Game of Life with all cells initially dead.
     ///
     /// The function will adjust the number of rows and columns based on the given
-    /// `cell_size` and will initialize the cells with random boolean values.
-    /// 
-    /// # Arguments 
+    /// `cell_size` and will initialize all cells to be dead (false).
+    ///
+    /// # Arguments
     ///
     /// * `height` - The height of the game grid.
```

Comment:
Maybe make it slightly clearer that ´height´ and ´width´ are the window dimensions in pixels. It took me a second to figure it out.

Reply:
Actually you made me realize of an issue: 

`height` and `width` refer to the **grid** dimensions (in pixels), not the **macroquad window** dimensions. So for example now you can have a 400x200 grid inside a 800x600 window (macroquad's default), which makes no sense. I'll fix this so that both grid and window dimensions match.